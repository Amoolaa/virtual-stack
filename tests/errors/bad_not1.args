"tests/errors/bad_not1.x2017"
