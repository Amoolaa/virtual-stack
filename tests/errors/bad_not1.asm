FUNC LABEL 0
    MOV STK A VAL 9
    NOT STK A
    PRINT STK A
    RET
