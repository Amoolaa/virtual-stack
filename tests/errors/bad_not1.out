NOT: argument must be register.
